main/models.py

```diff
@@ -20,6 +20,29 @@ def publish(self):
     def __str__(self):
         """Return title of Post.
 
-        This is useful if you need to get info only about title of Post object
+        This is used when rendering model instance as a string
         """
         return self.title
+
+    def approved_comments(self):
+        """Approving comments. Use for moderating."""
```

Comment:
`Approving` is Present Continuous Tense. Write it in imperative mode. Also you didn't describe an action this method does.